src/SFA.DAS.Forecasting.Application/Shared/Services/EmployerDataService.cs

```diff
@@ -69,5 +71,21 @@ IEmployerDatabaseService databaseService
                 SubmissionDate = levy.SubmissionDate,
             }).ToList();
         }
+
+        public async Task<List<long>> EmployersForPeriod(string payrollYear, short payrollMonth)
+        {
+            var accountIds = await _databaseService
+                .GetAccountIds(payrollYear, payrollMonth);
+
+            if (accountIds == null || !accountIds.Any())
+            {
+                _logger.Warn($"Not able to find any EmployerAccountIds, Year: {payrollYear}, Month: {payrollMonth}");
```

Comment:
Would be tempted to make this info rather than warning - as unlikely as it might be to happen